panel/src/panel/panel.js

```diff
@@ -67,7 +67,7 @@ export const states = [
 export default {
 	create(plugins = {}) {
 		// props
-		this.isLoading = false;
+		this.controller = null;
```

Comment:
`Suggestion` I would probably call it abortController to make instantly clear what this is controlling.